control-plane/agents/src/bin/core/tests/volume/switchover.rs

```diff
@@ -162,6 +162,20 @@ async fn old_nexus_delete_after_vol_destroy() {
         .start(cluster.node(0).as_str())
         .await
         .expect("failed to start container");
+    let mut count = 0;
```

Comment:
this kind of thing could be useful, please add a function to helpers.rs
See existing code where we panic based on timeout rather than iteration count.

Reply:
Sure

Reply:
Done